### PR TITLE
Upgrade Duplicity and Duply

### PR DESCRIPTION
**Pull request by cweagans**
I'm not sure if this is everything that needs done, but I couldn't see what else to change. I need the upgrade for Backblaze B2 support.

### CODE REVIEW

**Comment by Dr-Bean** [comment]
@romanlum @adrien-delhorme As the both of you have worked on duply and duplicity, could you please take a look at this? Thanks.

File: cross/duplicity/digests [hunk 1]
```diff
@@ -1,3 +1,3 @@
-duplicity-0.7.05.tar.gz SHA1 773513e95d788072c8bd5db7212a25ee247fe1af
-duplicity-0.7.05.tar.gz SHA256 1e53807d5de5bacbbe6c4eea2f313a16e9a63423316ae7b8597ee0d9e697f744
-duplicity-0.7.05.tar.gz MD5 b2f28a48aa392e81c740ac30d28e9250
+duplicity-0.7.05.tar.gz SHA1 08b7ee08e52205b73541829f83485b79f360ba2c
```
Comment (adrien-delhorme) [inline cross/duplicity/digests]: The name of the .tar.gz file still contains the previous version number

**Comment by adrien-delhorme** [comment]
Looks good to me, except the version number in the duplicity's digests file.

File: spk/duplicity/Makefile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 SPK_NAME = duplicity
-SPK_VERS = 0.7.05
+SPK_VERS = 0.7.06
 SPK_REV = 2
```
Comment (Dr-Bean) [inline spk/duplicity/Makefile]: If you update the changelog, you'll need to set SPK_REV as well; `3`.

Comment (cweagans) [inline spk/duplicity/Makefile]: Fixed.

File: cross/duplicity/digests (hunk 1 above)
Comment (cweagans) [inline cross/duplicity/digests]: Whoops! Fixed.

**Comment by cweagans** [comment]
Those issues have been fixed.

**Comment by Dr-Bean** [comment]
Let's slap a ready-to-merge tag on it then ;)

**Comment by Dr-Bean** [comment]
For next time, please test the changes before opening a PR, if for no other reason than to confirm the new functionality works as you expect.
In this case, the included patches were broken due to upstream changes as well, so that should have been part of the PR too.

**Pull request merged**